### ISSUE TITLE
{Suggestion}: Add es_es language support (Castillian Spanish)

### ISSUE THREAD

**Issue by Daralexen05** [Legodamen/The-John-Mod-Reborn#11]
The repository is empty, so I am unable to create a pull request. I made a Castilian Spanish translation of what I could understand in the en_us.json file of this mod months ago. This is for general ease of play and immersion. It is jarring, to say the least, to constantly switch my keyboard language to search for items in JEI from this mod, or to hear The Hunted crying for me in English when the rest of my game is in Spanish.

**Comment by Legodamen** [comment]
yeah sure, if you have discord join our discord group. We can talk there



https://discord.gg/twwMj3UrPc
